src/azure-cli/azure/cli/command_modules/appconfig/_featuremodels.py

```diff
@@ -306,60 +845,142 @@ def map_keyvalue_to_featureflagvalue(keyvalue):
             for client_filter in client_filters:
                 # If there is a filter, it should always have a name
                 # In case it doesn't, ignore this filter
-                name = client_filter.get(FeatureFlagConstants.FILTER_NAME)
+                lowercase_filter = {k.lower(): v for k, v in client_filter.items()}
+                name = lowercase_filter.get(FeatureFlagConstants.NAME)
```

Comment:
nit: Can we have explicit default values here like in other places?

Reply:
Added default values